rust/sealevel/programs/hyperlane-sealevel-token-collateral/Cargo.toml

```diff
@@ -15,15 +15,19 @@ num-traits.workspace = true
 solana-program.workspace = true
 spl-associated-token-account.workspace = true
 spl-noop.workspace = true
-spl-token-2022.workspace = true # FIXME Should we actually use 2022 here or try normal token program?
+spl-token-2022.workspace = true               # FIXME Should we actually use 2022 here or try normal token program?
```

Comment:
nit: rm the whitespace here?

Reply:
Discussed with @daniel-savu to leave it as a part of TOML formatting